src/hypothesis/internal/charmap.py

```diff
@@ -241,7 +272,7 @@ def _category_key(exclude, include):
     If include is None then default to including all categories.
     Any item in include that is not a unicode character will be excluded.
 
-    >>> _category_key(exclude=['So'], include=['Lu', 'Me', 'Cs', 'So', 'Xx'])
```

Comment:
So we previously had a docs item with an invalid category? In that case I'm afraid to say that I really think the change should be a warning not an error.

Reply:
I've come to a similar view - it's not user-visible docs, but does indicate that this is an easy mistake to make.